contrib/renew-client-cert

```diff
@@ -39,7 +39,20 @@ rotate() {
 	if [ -n "$cert_ids" ]; then
 		extra_args="$extra_args --pkcs11-cert-ids=$cert_ids"
 	fi
-	exec $fioconfig -c $SOTA_DIR renew-cert $extra_args $est_server $rotation_id 
+	# fioconfig's http logic is already doing back off logic with intervals:
+	#  1, 2, 5, 13, 30 (seconds)
+	# We should sleep in *minutes* given
+	for x in 0 1 2 5 13 ; do
+		if [ $x -ne 0 ] ; then
+			echo "Rotation failed. Trying again in ${x} minutes ..."
+		fi
+		sleep ${x}m
+		if $fioconfig -c $SOTA_DIR renew-cert $extra_args $est_server $rotation_id  ; then
+			exit 0
+		fi
+	done
+	echo "FAILED to rotate certificates restarting fioconfig daemon as a last resort"
+	systemctl restart fioconfig
```

Comment:
Is there a way for fioconfig to fail instead?

Just thinking if this will indeed cause the right behavior in systemd as the process is restarting itself after it was initialized by systemd (there are restart conditions that can be applied in systemd for x amount of failures for example, and this restart will probably zero on every run).